Tests/SwiftPrettyPrintTests/Core/URLColoringWithDefaultColorThemeTests.swift

```diff
@@ -20,54 +20,20 @@ import Curry
 /// - SeeAlso:
 /// [ANSI escape code - Wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code)
 class URLColoringWithDefaultColorThemeTests: XCTestCase {
-
     let describerWithDefaultColorTheme = PrettyDescriber(formatter: SinglelineFormatter(), theme: .default)
-    let exampleURL = "https://example.com"
 
-    override func setUpWithError() throws {}
+    override func setUp() {}
 
-    override func tearDownWithError() throws {}
```

Comment:
`setUpWithError()` and `tearDownWithError()` are not included in Swift5.1 in Linux env.